tests/unit/StaffNotation.spec.ts

```diff
@@ -208,6 +210,41 @@ describe('StaffNotation', () => {
     wrapper.destroy()
   })
 
+  it('renders the staff notation dialog for Admin Freeze', async () => {
+    // set store specifically for this test
+    store.state.entityType = 'SP'
+    store.state.entityState = 'ACTIVE'
+
+    const wrapper = mount(StaffNotation, { vuetify, store })
+
+    // open menu
+    await wrapper.find('.menu-btn').trigger('click')
+    expect(wrapper.vm.$data.expand).toBe(true)
+
+    // find and click respective item
+    await wrapper.find('.v-list-item[data-type="admin-freeze"]').trigger('click')
+    await Vue.nextTick()
+
+    // verify flag
+    expect(wrapper.vm.$data.isAddingAdministerFreeze).toBeTruthy()
```

Comment:
"Truthy" is almost anything. It's better to check `toBe(true)`, which is an exact match.

(To all devs: same for "toBeFalsy": could be 0, false, empty string, null, undefined. That is not very specific. We should be specific since we should know exactly what the value is.)